mobile/src/home-view.js

```diff
@@ -156,13 +155,11 @@ export default class HomeView extends Component {
   }
 
   trimWhiteSpaceItem = () => {
-    const {currentItem} = this.state
-    const editingItem = {
-      ...currentItem,
-      description: currentItem.description.trim(),
-      lastLocation: currentItem.lastLocation.trim(),
-      currentLocation: currentItem.type === 'found' ? currentItem.currentLocation.trim() : undefined
-    }
+    const { currentItem } = this.state
+    let editingItem = currentItem
```

Comment:
This is modifying state without calling `setState()` again

Reply:
Correct but the variable current item always gets reset to {} right after this function is used.